warehouse/templates/rss/project_updates.xml

```diff
@@ -0,0 +1,14 @@
+{% extends "base.xml" %}
+{% block title %}Latest releases of the {{ project.name }} package{% endblock %}
```

Comment:
Nitpick: this should probably say "project", not package. It would be good to put the project name in quotes too.